nilearn/maskers/tests/test_multi_nifti_masker.py

```diff
@@ -25,7 +25,15 @@ def test_auto_mask():
     masker = MultiNiftiMasker(mask_args=dict(opening=0))
     # Check that if we have not fit the masker we get a intelligible
     # error
-    pytest.raises(ValueError, masker.transform, [[img, ]])
+    pytest.raises(
+        ValueError,
+        masker.transform,
+        [
+            [
+                img,
+            ]
+        ],
+    )
```

Comment:
Anyway to avoid this?

Also note for future refactoring, these should use context manager

Reply:
removing the coma is should tell black to put that on oneline

let me fix this

Reply:
yup that dit it